R/tm_g_scatterplotmatrix.R

```diff
@@ -202,7 +202,15 @@ tm_g_scatterplotmatrix <- function(label = "Scatterplot Matrix",
 
   checkmate::assert_multi_class(pre_output, c("shiny.tag", "shiny.tag.list", "html"), null.ok = TRUE)
   checkmate::assert_multi_class(post_output, c("shiny.tag", "shiny.tag.list", "html"), null.ok = TRUE)
-  checkmate::assert_list(decorators, "teal_transform_module", null.ok = TRUE)
+
+  if (checkmate::test_list(decorators, "teal_transform_module", null.ok = TRUE)) {
+    decorators <- if (checkmate::test_names(names(decorators), subset.of = c("default", "plot"))) {
+      lapply(decorators, list)
+    } else {
+      list(default = decorators)
+    }
+  }
```

Comment:
@averissimo what you think about wrapping that up in a function? looks like it's gonna be repeated across modules

Reply:
:heart: this! Will do after cleaning up all modules

Reply:
Keeping this conversation open until that is done